ArcGISRuntimeSDKQt_CppSamples/Analysis/StatisticalQuery/StatisticalQuery.cpp

```diff
@@ -89,9 +89,9 @@ void StatisticalQuery::componentComplete()
     {
       StatisticRecord* record = iter.next(&parent);
       const QVariantMap& statsMap = record->statistics();
-      for (auto it = statsMap.cbegin(); it != statsMap.cend(); ++it)
+      for (const auto& it : statsMap.toStdMap())
```

Comment:
As far as I am aware `QVariantMap` has an iterator type. Why are we converting in this instance? This goes foe the other instances too.

Reply:
Wait I see - the iterator dereference is the value type only, and you need key/value pairs. I wonder if this change is appropriate in this case.

Reply:
Oh I think you are right and that this was only supposed to apply to list model types and I just got carried away.

Reply:
the main purpose of the issue was to scan the repo to replace any c style for loops with range based for loops for list model types, since that is the new support James recently added. Looks like maybe we didn't have any instances of this in the samples? If there are other improvements to make, I'm ok with having those go in. I'll defer to @anmacdonald on the review comments

Reply:
I would back out this change. Converting this to a std::map should not be needed.

Reply:
Thanks for the input! I will back this up to what was there prior.